src/main/java/eu/deltasource/internship/hotelmanagementsystem/Room.java

```diff
@@ -0,0 +1,191 @@
+package eu.deltasource.internship.hotelmanagementsystem;
+
+/**
+ * Class Room has 5 private members
+ * methods - setters, getters, constructors and other methods
+ */
+
+import eu.deltasource.internship.hotelmanagementsystem.hotel.service.domain.commodities.AbstractCommodity;
+import eu.deltasource.internship.hotelmanagementsystem.hotel.service.domain.commodities.Bed;
+import eu.deltasource.internship.hotelmanagementsystem.hotel.service.domain.commodities.Booking;
+
+import java.time.LocalDate;
+import java.util.*;
+
+/**
+ * Represents a room in a hotel
+ */
+public class Room {
+
+    private int roomNum;
+    private short capacity;
+    private Set<AbstractCommodity> commodities;
+    private Set<LocalDate> maintenanceDates;
+    private Set<Booking> bookings;
+
+    /**
+     * Parametrized constructor
+     *
+     * @param roomNum          ID of the room
+     * @param commodities      set of commodities
+     * @param maintenanceDates set of
+     * @param bookings         set of bookings
+     */
+    public Room(int roomNum, Set<AbstractCommodity> commodities,
+                Set<LocalDate> maintenanceDates, Set<Booking> bookings) {
+        this.roomNum = roomNum;
+        this.commodities = commodities;
+        this.maintenanceDates = maintenanceDates;
+        this.bookings = bookings;
```

Comment:
In general let's initialize the collections in our objects, when we create them, there is no need to pass in new collection instances when we create the objects.